packages/mds-event-server/index.ts

```diff
@@ -46,7 +45,11 @@ export const EventServer = <TData, TResult>(
     try {
       const event = parseCloudEvent(req)
       await logger.info('PARSE Cloud Event', 'BODY:', req.body, 'HEADERS:', req.headers, 'EVENT:', event.format())
-      const result = await handler(event.getType(), event.getData(), event)
+      // fixme: regex failed me; event.getType().replace(/^`${tenantId}`./, '')
```

Comment:
I have some comments/recommendations on the changes to this module, but I'm AFK.